src/Devolutions.MacOS.Avalonia.Theme/Devolutions.MacOS.Avalonia.Theme.csproj

```diff
@@ -1,6 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
+        <Version>1.0.0.0</Version>
```

Comment:
Perhaps at this stage I'd be more comfortable with `0.0.0.1` or something like that ... 🙂